frontend/public/components/start-guide.tsx

```diff
@@ -22,105 +22,103 @@ export const OpenShiftGettingStarted = connect(createProjectMessageStateToProps)
+        component="a"
+        href="/command-line-tools"
+      >
+        {t('public~Download command-line tools')}
+      </Button>,
+      <Button key="visit-docs" variant={ButtonVariant.link} component="a" href={openshiftHelpBase}>
+        {t('public~View documentation')}
+      </Button>,
+    ];
+
     return (
-      <>
+      <ConsoleEmptyState
+        icon={CubesIcon}
+        title={t('public~Hello, world')}
+        primaryActions={primaryActions}
+        secondaryActions={secondaryActions}
+      >
         {canCreate ? (
-          <p>
-            {t(
-              'public~OpenShift helps you quickly develop, host, and scale applications. To get started, create a project for your application.',
-            )}
-          </p>
+          <p>{t('public~To get started, create a project for your application.')}</p>
         ) : (
-          <p>
-            {t(
-              "public~OpenShift helps you quickly develop, host, and scale applications. To get started, you'll need a project. Currently, you can't create or access any projects.",
-            )}
-            {!createProjectMessage && (
-              <>&nbsp;{t("public~You'll need to contact a cluster administrator for help.")}</>
+          <>
+            <p>
+              <Trans t={t} ns="public">
+                To get started, you'll need a project. Currently, you can't create or access any
+                projects.
+              </Trans>
+              {!createProjectMessage && (
+                <>&nbsp;{t("public~You'll need to contact a cluster administrator for help.")}</>
+              )}
+            </p>
+            {createProjectMessage && (
+              <p className="co-pre-line">
+                <LinkifyExternal>{createProjectMessage}</LinkifyExternal>
+              </p>
             )}
-          </p>
+          </>
         )}
-        {createProjectMessage && (
-          <p className="co-pre-line">
-            <LinkifyExternal>{createProjectMessage}</LinkifyExternal>
-          </p>
-        )}
-        <p>
-          <Trans t={t} ns="public">
-            To learn more, visit the OpenShift{' '}
-            <ExternalLink href={openshiftHelpBase}>documentation</ExternalLink>.
-          </Trans>
-        </p>
-        <p>
-          <Trans t={t} ns="public">
-            Download the <Link to="/command-line-tools">command-line tools</Link>
-          </Trans>
-        </p>
-        {canCreate ? (
-          <Button
-            variant="link"
-            onClick={() =>
-              createNamespaceOrProjectModal({
-                onSubmit:
-                  perspective !== 'admin'
-                    ? (project: K8sResourceKind) => {
-                        setActiveNamespace(project.metadata?.name);
-                      }
-                    : undefined,
-              })
-            }
-          >
-            {t('public~Create a new project')}
-          </Button>
-        ) : null}
-      </>
+      </ConsoleEmptyState>
```

Comment:
you need to run `yarn i18n` to update the strings here.. or maybe we should keep the drab empty state for 4.19? @TheRealJon

Reply:
Update: After the discussion, this should be back-ported. The translation files should also be included. Since this is a manual back port PR, I will make sure all the necessary and related changes are included.